pages/more-info.tsx

```diff
@@ -0,0 +1,28 @@
+import { withPageAuthRequired } from '@auth0/nextjs-auth0';
+import * as data from '../data/power_card_data.json';
```

Comment:
this seems to be giving the following error: 
```
./pages/more-info.tsx
Should not import the named export 'appliances'.'map' (imported as 'data') from default-exporting module (only default export is available soon)
```
instead, it should suffice to just do: 
```suggestion
import data from '../data/power_card_data.json';
```

furthermore, I don't really think `data` is the best name for this. perhaps `content` would be more accurate, as this is really the content we're displaying on this page. 

finally, I don't think this JSON file should be in the `data` folder, as that folder currently just holds our data that we feed into the mock API. lets create a new folder called `/content`, and toss this file in there. perhaps even rename it to `power-card-content.json`. for a future tip, we seem to have settled on `kebab-case` rather than `snake_case` for our non-react component file names, and we'd very much like to keep that consistent.